apps/web/src/app/layout.tsx

```
@@ -45,6 +48,26 @@ export default function RootLayout({
         </AppProviders>
         <div id="modal-root" />
         <div id="toast-root" />
+        {beusableId && (
+          <Script
+            id="beusable-rum"
+            strategy="afterInteractive"
+            dangerouslySetInnerHTML={{
+              __html: `
+                (function(w, d, a){
+                  w.beusablerumclient = {
+                      load : function(src){
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)